EngineLayer/Psm.cs

```diff
@@ -401,14 +401,14 @@ internal void Add(Psm psmParent)
         private Tuple<string, double?> Resolve(IEnumerable<double> enumerable)
         {
             double doubleTol = 1e-6;
-            var list = enumerable.ToList();
+            var list = enumerable.Distinct().ToList();
```

Comment:
I just think for the stop and start or's it wouldn't be easy to convey the info simply without listing the protein each time